core/client/ui/settings-main.js

```diff
@@ -7,6 +7,7 @@ Template.settingsMain.onCreated(() => {
 Template.settingsMain.helpers({
   activePage() { return Session.get('activeSettingsPage') || getDefaultActivePage(); },
   allowProfileEdition() { return Meteor.settings.public.permissions.allowProfileEdition; },
+  helpURL() { return Meteor.settings.public.lp.helpURL; },
```

Comment:
For later, I saw that there is a helper that allows you to access settings directly from a template: `Template.registerHelper('settings', () => Meteor.settings);` so we should be able to do `settings.public.lp.helpURL` from the Blaze files.

Reply:
Yeah we should definitely refactor these. :)

#126